src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -156,11 +161,56 @@ public boolean isPatch(Method method) {
     return apiaryConfig.getHttpMethod(method.getName()).equals("PATCH");
   }
 
-  // Returns default string. If ApiaryConfig specifies a pattern, default string for that pattern
-  // is returned. Otherwise, returns empty string.
+  /*
+   * Returns language-specific syntax of string literal with given value.
+   */
+  public String stringLiteral(String value) {
+    return "\"" + value + "\"";
+  }
+
+  /*
+   * Returns language-specific syntax of line ending with given value.
+   */
+  public String lineEnding(String value) {
+    return value + ";";
+  }
+
+  /*
+   * Returns language-specific syntax of line commented with given comment string.
+   */
+  public String lineComment(String line, String comment) {
+    return line + "  // " + comment;
+  }
+
+  /*
+   * Returns default string (to end of line) for a type's string field. If ApiaryConfig specifies a
+   * string format, returns corresponding default string with inline comment. Otherwise, if
+   * ApiaryConfig specifies a pattern, returns corresponding default string. Otherwise, returns
+   * empty string.
+   */
   protected String getDefaultString(Type type, Field field) {
```

Comment:
Now that we are doing this, let's put the formatting in individual languages' templates.

The way I'm imagining this, `getDefaultString` (or something similarly named) can return both the string and the comment. The template can figure out what to do with them.